MHTimer.js

```diff
@@ -220,6 +220,7 @@ function Main() {
 
                 // Use one timeout per timer to manage default reminders and announcements.
                 timers_list.forEach(timer => scheduleTimer(timer, announcables));
+                resetRH(); // Default relic hunter to appropriate nothingness
```

Comment:
I now have it querying the sources separately on startup. Testing it a bit before I even think about merging it since nobody is hunting in CSS and there's currently a situation that MHCT doesn't know where the RH is.